src/Extensions/Hosting/Certificates/CertificateReader.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.Extensions.Abstractions;
+
+namespace Microsoft.Omex.Extensions.Hosting.Certificates
+{
+	internal class CertificateReader : ICertificateReader
+	{
+		private readonly ICertificateStore m_certificateStore;
```

Comment:
>private [](start = 2, length = 7)

Do we keep spacing between private member variables?

Reply:
I don't think we enforce empty lines between private members especially since they don't have comments, to me it makes sense to combine them without spaces, what do you think?